### DISCUSSION TITLE
DRM removed for MOBI but not EPUB

### DISCUSSION THREAD

**Discussion by DaddySwells** [noDRM/DeDRM_tools#198] [Q&A]
Hi, 

I'm able to successfully DeDRM AZW3 books when converting to MOBI and can send to my Kindle using the "Send to Kindle" service. I then get an email from Amazon telling me it's decommissioning MOBI as an accepted format so move to EPUB. 

Calibre seems to successfully convert to EPUB (the job completes) but when I send it to my Kindle using "Send to Kindle" I receive an email from Amazon saying it was unsuccessful and it is likely to be because the file is protected. I have removed and readded my EInk serial number as a attempted problem solve. 
 
Any help gratefully received ;-)

MacOS 13
Calibre 6.7.1
DeDRM 10.0.3

**Comment by ElleKayEm** [comment]
If you can convert at all, the DRM is removed.  The problem may be that Amazon is very particular about the formatting of the epub.  I've seen some discussion about this over at MobileRead.com.  You might try looking or asking about it there.

**Comment by DaddySwells** [comment]
Thanks for the suggestion. After reading a few threads I tried the following and it seems to have worked:

1 - Change the input character encoding to utf-8
2 - Change the EPUB output version to 2

I don't know if one or both are required but now it works again

**Comment by JSWolf** [comment]
> Hi,
> 
> I'm able to successfully DeDRM AZW3 books when converting to MOBI and can send to my Kindle using the "Send to Kindle" service. I then get an email from Amazon telling me it's decommissioning MOBI as an accepted format so move to EPUB.
> 
> Calibre seems to successfully convert to EPUB (the job completes) but when I send it to my Kindle using "Send to Kindle" I receive an email from Amazon saying it was unsuccessful and it is likely to be because the file is protected. I have removed and readded my EInk serial number as a attempted problem solve.
> 
> Any help gratefully received ;-)
> 
> MacOS 13 Calibre 6.7.1 DeDRM 10.0.3

Two things to do.

1. Use the Modify ePub plugin to set the ePub to UTF-8.
2. Load the ePub into the Calibre editor and check for errors using the error checker and epubcheck (editor plugin). 

Once you've done both, should be good to go. However, the optimal solution is to forget send-to-Kindle and just load the KF8 eBook to your Kindle via USB. Forget syncing and email.
